packages/in_app_purchase/in_app_purchase_storekit/lib/src/in_app_purchase_storekit_platform.dart

```diff
@@ -251,11 +251,18 @@ class InAppPurchaseStoreKitPlatform extends InAppPurchasePlatform {
   @Deprecated('Use countryCode')
   Future<String?> getCountryCode() => countryCode();
 
-  /// Turns on StoreKit2. You cannot disable this after it is enabled.
-  /// This can only be enabled if your device supports StoreKit 2.
+  /// StoreKit 2 is now the default.
+  @Deprecated('StoreKit 2 is now the default')
   static Future<bool> enableStoreKit2() async {
+    _useStoreKit2 = true;
+    return true;
+  }
+
+  /// Call this before `registerPlatform` to conditionally support StoreKit 1 for
+  /// devices below iOS 15.
+  static Future<bool> enableStoreKit1() async {
     _useStoreKit2 = await SKRequestMaker.supportsStoreKit2();
-    return _useStoreKit2;
+    return !_useStoreKit2;
```

Comment:
why does a function called `enableStoreKit1` check `supportsStoreKit2()`, rather than enabling store kit 1?

Reply:
it should just be the negation of it, I only wanted them to be able to enable storekit 1 on devices that dont support storekit 2. But maybe I should just go ahead and allow them to switch back to storekit 1 no matter what

Reply:
im confused. my understanding is that, a function called "enableStoreKit1" should "enable store kit 1". or am i missing something?

Reply:
Originally I wanted it to only enable store kit 1 if the device doesn't support store kit 2. But that seems kind of unintuitive so I changed it